lib/WP_Auth0_LoginManager.php

```diff
@@ -551,5 +529,15 @@ protected function query_vars( $key ) {
     if ( isset( $_REQUEST[$key] ) ) return $_REQUEST[$key];
     return null;
   }
-
+  
+  /**
+   * DEPRECATED 3.4.1
+   *
+   * @param $userinfo
+   * @param $id_token
+   */
+  private function dieWithVerifyEmail( $userinfo, $id_token = '' ) {
+    trigger_error( __( 'Method dieWithVerifyEmail is deprecated.', 'wp-auth0' ), E_USER_DEPRECATED);
```

Comment:
I don't fully understand the `trigger_error` method. Is it just logging an error on the console or exploding over the air? If the latter then this is no deprecation. This is breaking the api 🤕 . Old methods must remain the same, working without changing their signature at all. Normally we achieve this by re-routing the call to the new method that will *eventually* replace it. On a side note, we prefer to handle deprecations on a separate PR so the changelog is clearer. Would you mind rolling back this change?

Reply:
@lbalmaceda - `trigger_error` is just that, triggers an error of a certain type. Depending on the error type and the environment, it will be handled different ways. In the case of a deprecated error like this one, it's either displayed or not, won't stop the process from continuing. 

Happy to do whatever is prudent here but the idea here is that the function still exists and can be called as based but does the right thing with a soft error. Signature is essentially the same, just defaulting to a blank `$id_token` and the replacement function is called with the `$userinfo` it needs. The chances that this function is being used in the wild is very small but, if so, this should keep folks running. 

Let me know if you still want me to back it out and how to handle going forward!